examples/go/transfer_basic/main.go

```diff
@@ -64,10 +64,40 @@ func transfer(api *gsrpc.SubstrateAPI, senderSeed string, receiver string, amoun
 		return fmt.Errorf("cannot get latest storage:%w", err)
 	}
 
+	// nonce := uint32(accountInfo.Nonce)
+	// options := types.SignatureOptions{
+	// 	BlockHash:          genesisHash,
+	// 	Era:                types.ExtrinsicEra{IsMortalEra: false},
+	// 	GenesisHash:        genesisHash,
+	// 	Nonce:              types.NewUCompactFromUInt(uint64(nonce)),
+	// 	SpecVersion:        rv.SpecVersion,
+	// 	Tip:                types.NewUCompactFromUInt(0),
+	// 	AppID:              types.NewUCompactFromUInt(uint64(0)),
+	// 	TransactionVersion: rv.TransactionVersion,
+	// }
+	latestHash, err := api.RPC.Chain.GetBlockHashLatest()
+	if err != nil {
+		panic(fmt.Sprintf("cannot create storage key:%w", err))
+	}
+	latestBlock, err := api.RPC.Chain.GetBlockLatest()
+	if err != nil {
+		panic(fmt.Sprintf("cannot create storage key:%w", err))
+	}
+	var e types.ExtrinsicEra
+	e.IsMortalEra = true
+	number := latestBlock.Block.Header.Number
+	second := types.U64(number) % 128
+	fmt.Println(number)
+	// mortl := types.NewMortalEra(number, types.U64(256))
```

Comment:
Is this comment relevant?

Reply:
Yeh, there are two ways to create a mortal transaction and logging it there as a ref for future if someone needs it

Reply:
What do you think about having two examples instead using one example for two different things?

Reply:
done!